src/Ookii.Dialogs.Wpf/Ookii.Dialogs.Wpf.csproj

```diff
@@ -1,51 +1,68 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{D01B1D20-8F5B-4834-8E5C-C7EC6DD587D4}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Ookii.Dialogs.Wpf</RootNamespace>
+    <TargetFrameworks>netcoreapp3.1;net45</TargetFrameworks>
```

Comment:
Does this work fine for you with VS 2019 in this order? I remember having issues with this in some projects where it was necessary to declare `net45` first, and `netcoreapp3.1` after. i.e.

```xml
<TargetFrameworks>net45;netcoreapp3.1</TargetFrameworks>
```

Reply:
It works on my side. Although, I have .NET Core 3.1 SDK already installed. It is possible to specify which version to compile in Visual Studio. But, as a failsafe, it would be a good idea.